test/unit/utils/users.test.js

```diff
@@ -0,0 +1,85 @@
+/**
+ * This eslint rule is disabled because of https://github.com/nodesecurity/eslint-plugin-security/issues/21
+ * It gives linting errors in testing the DB data with keys from fixtures
+ */
+/* eslint-disable security/detect-object-injection */
```

Comment:
On which lines were you getting the error?
Since it says `security`, let us not avoid it. We should try to fix it.

Reply:
Yes ok. I'll re-check where the error is occurring

Reply:
> On which lines were you getting the error?
> Since it says `security`, let us not avoid it. We should try to fix it.

I checked the files. Seems to disable the eslint-rule was unnecessary as the code is working fine without it.  I must have encountered the error earlier but forget to check if it has been resolved.